route.go

```diff
@@ -532,6 +532,30 @@ func (r *Route) URLPath(pairs ...string) (*url.URL, error) {
 	}, nil
 }
 
+// GetPathTemplate and GetHostTemplate returns the template used to match against for the route
```

Comment:
`// GetPathTemplate and GetHostTemplate return the template used to build the route match.`